lib/Sema/TypeCheckAttr.cpp

```diff
@@ -3849,6 +3839,7 @@ bool checkIfDifferentiableProgrammingEnabled(
 
 llvm::Expected<IndexSubset *> DifferentiableAttributeTypeCheckRequest::evaluate(
     Evaluator &evaluator, DifferentiableAttr *attr) const {
+#if 0
```

Comment:
what are all the `#if 0`s for?

Reply:
Oops, those changes were not intentional. I've reverted them, the latest commit just addresses your review feedback.